User tutorials update (#698)
kyma-project/lifecycle-manager: docs/modularization.md

Modules are the next generation of components in Kyma that are available for local and cluster installation.

> Modules are no longer represented by a single helm-chart, but instead are bundled and released within channels through a [ModuleTemplate custom resource (CR)](../api/v1beta2/moduletemplate_types.go), a unique link of a module, and its desired state of charts and configuration, and a channel.

Reviewer: ```suggestion
Modules are no longer represented by a single helm-chart, but instead are bundled and released within channels through a [ModuleTemplate custom resource (CR)](../api/v1beta2/moduletemplate_types.go), a unique link of a module, and its desired state of resources and configuration, and a channel.
```

Lifecycle Manager manages clusters using the [Kyma CR](../api/v1beta2/kyma_types.go). The CR defines the desired state of modules in a cluster. With the CR you can enable and disable modules with domain-specific functionality with additional configuration.

The modules themselves are built and distributed as OCI artifacts. The internal structure of the artifact conforms to the [Open Component Model](https://ocm.software/) scheme version 3. Modules contain an immutable layer set of a module operator deployment description and its configuration.

![Kyma Module Structure](/docs/assets/kyma-module-template-structure.svg)

> If you use Kyma [CLI](https://github.com/kyma-project/cli), you can create a Kyma module by running `kyma alpha create module`. This command creates a component descriptor in the configured descriptor path (`./mod` as default) and packages all the contents on the provided path as an OCI artifact. Refer to the `kyma alpha create module --help` section to learn more about the module structure and how it is created. You can also use the module's inbuilt auto-detection of [Kubebuilder](https://kubebuilder.io) projects to easily bundle your module with little effort.

Reviewer: ```suggestion
If you use Kyma [CLI](https://github.com/kyma-project/cli), you can create a Kyma module by running `kyma alpha create module`. This command packages the module's contents on the provided path as an OCI artifact and pushes the artifact to the provided OCI registry. Use the `kyma alpha create module --help` command to learn more about the module structure and how it is created. You can also use the CLI's auto-detection of [Kubebuilder](https://kubebuilder.io) projects to easily bundle your module with little effort.
```
